packages/gensx-react/src/hooks/use-gensx.ts

```diff
@@ -200,10 +227,62 @@ export function useWorkflow<TInputs = unknown, TOutput = unknown>(
             setInProgress(false);
             onError?.(event.error);
             break;
+
+          case "external-tool":
```

Comment:
will we use this case to handle human in the loop or do we need a separate case for that?

Reply:
Human in the loop is upto the caller. They provide a `trigger` function which gets the callback URL as an argument, and inside that function they  push that callback URL to whatever system they need.